crates/core/c8y_api/src/http_proxy.rs

```diff
@@ -128,22 +129,29 @@ impl C8yMqttJwtTokenRetriever {
         // Ignore errors on this connection
         mqtt_con.errors.close();
 
-        mqtt_con
-            .published
-            .publish(mqtt_channel::Message::new(
-                &Topic::new_unchecked("c8y/s/uat"),
-                "".to_string(),
-            ))
-            .await?;
-
-        let token_smartrest =
-            match tokio::time::timeout(Duration::from_secs(10), mqtt_con.received.next()).await {
-                Ok(Some(msg)) => msg.payload_str()?.to_string(),
+        tokio::time::sleep(Duration::from_millis(20)).await;
+        for _ in 0..3 {
+            mqtt_con
+                .published
+                .publish(mqtt_channel::Message::new(
+                    &Topic::new_unchecked("c8y/s/uat"),
+                    "".to_string(),
+                ))
+                .await?;
+            debug!("JWT token requested");
```

Comment:
Why not `info`?

Reply:
Yeah I think having `info` level would be helpful as the log level is set to `info` by default.

Reply:
Done